source/upgrade/important-upgrade-notes.rst

```diff
@@ -43,6 +43,8 @@ Important Upgrade Notes
 |                                                    |                                                                                                                                                                  | 
 |                                                    | Added a new configuration setting ``DataRetentionSettings.RetentionIdsBatchSize``, which allows admins to configure how many batches of IDs will be fetched at   |
 |                                                    | a time when deleting orphaned reactions. The default value is 100.                                                                                               |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Modifield OAuth/SAML flows to include ``desktop_login`` where applicable. This change makes Desktop v5.2 and below incompatible with server v9.1 and later.      |
```

Comment:
To confirm, the server is compatible on v5.3 and later?

Reply:
Correct